pkg/metrics/controller_metrics.go

```diff
@@ -21,6 +21,7 @@ import (
 )
 
 var (
+	distributionSeconds       = []float64{0, 1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384}
```

Comment:
nit: I would use a different name or not use a variable.

I think this boundaries only fits your metrics.

Reply:
Thanks for the update I would rename it to `stateDurationSeconds`.